### PR TITLE
Story Hierarchy - keyboard accessibility

### PR DESCRIPTION
**Pull request by igor-dv**
Issue: #151 

## What I did
I've added accessibility support to the stories tree.
storybook_hierarchy_react-treebeard-keyboard

## Next Steps
Support arrow keys to navigate on tree

## How to test
run `cra-kitchen-sink`  and press `tab`, `shit + tab` and `enter` (or any other OS alternative =/ ) when focus is on the tree.

### CODE REVIEW

**Comment by UsulPro** [comment]
@igor-dv it's cool!

but did you consider to improve the existing `Ctrl-Shift-Left/Right` shortcuts?

a bit more details about what I mean:

#981 PR introduces one more UI API for jumping to next/prev storyKind

I guess we can use it with hierarchy as well just need to add keyboard shortcuts

advantage: after merging it we can use the same way of navigation both by keyboard and by touch screen on mobile devices

/cc @shilman

**Comment by igor-dv** [comment]
@UsulPro - Great feature, would like to see it live !

What I tried to do here is to support the previous stories functionality - basic navigation with `tab` (actually in the previous navigation pressing on enter did nothing). I think that the core menu and a tab navigation don't collide one with each other, moreover IMO the core menu should support basic accessibility too =)

I will try to improve the existing `Ctrl-Shift-Left/Right` shortcuts in the next PR =)

What BTW prevents #981 from being merged ?

**Comment by UsulPro** [comment]
Good point 👍 

> support the previous stories functionality - basic navigation with tab 

I'm just not sure is it intentional feature or "side effect". I know some people don't like this blue frames #1293 Maybe somebody could clarify?

related to  #981 I guess I need to find some time to fix the issues requested by review (and maybe it needs one more review as well)

**Comment by igor-dv** [comment]
Yep, this blue box is not a big art =) But it could be styled appropriately (thinner and not blue for example)

**Comment by igor-dv** [comment]
Review will be appreciated =)

File: lib/ui/src/modules/ui/components/left_panel/stories_tree/tree_decorators.js
Comment (danielduan) [inline lib/ui/src/modules/ui/components/left_panel/stories_tree/tree_decorators.js]: these things inside the shape should probably all be `.isRequired` from the looks of how they're used.

File: lib/ui/src/modules/ui/components/left_panel/stories_tree/tree_decorators.js
Comment (danielduan) [inline lib/ui/src/modules/ui/components/left_panel/stories_tree/tree_decorators.js]: looks like `node.type` and `.children` could be checked as well

**Review by danielduan — APPROVED** [review]
looks good, thanks for adding a11y

**Review by shilman — APPROVED** [review]
@igor-dv this is really great!!! going the extra mile!

**Pull request merged**
